docs/guides/docs/running-a-node/running-a-local-node.mdx

```diff
@@ -42,48 +42,59 @@ forc deploy --default-signer --node-url 127.0.0.1:4000/v1/graphql
 To modify the initial state of the chain, you must configure the `state_config.json` file in your chain configuration folder.
 Here is an example of what that looks like using version {props.fuelCoreVersion} of `fuel-core`:
 
-<ConditionalContent versionSet={props.versionSet} showForVersions={["default", "nightly"]}>
-  Here is the [specific configuration](https://github.com/FuelLabs/fuel-core/tree/9fddeccb4d112c148f6793bc3d21131a13778a25/bin/fuel-core/chainspec/testnet)
+Here is the [specific configuration](https://github.com/FuelLabs/chain-configuration/tree/master/local)
 
```

Comment:
Can you add a section to clone the repository as the default option?

```
For simplicity, clone the [repository](https://github.com/FuelLabs/chain-configuration/tree/master) into the directory of your choice.

When using the `--snapshot` flag later, you can replace `./your/path/to/chain_config_folder` with the repository you just cloned `./chain-configuration/ignition/`.
```

Reply:
Make sure you change this to local and use the same thing during the ignition part